lib/health_web/pow/messages.ex

```diff
@@ -0,0 +1,19 @@
+defmodule HealthWeb.Pow.Messages do
+  use Pow.Phoenix.Messages
+  use Pow.Extension.Phoenix.Messages, extensions: [PowResetPassword]
+
+  import HealthWeb.Gettext
+
+  # For a list of messages:
+  # https://github.com/danschultzer/pow/blob/master/lib/pow/phoenix/messages.ex
+
```

Comment:
These 3 messages are `nil` by default, but I felt they were missing. I wanted to play with pow.

Reply:
Lol... I added my own too before... we may have a conflict

Reply:
Yea i just discovered this. Don't worry I deleted yours :)

Reply:
Haha